.github/workflows/coverage.yml

```diff
@@ -67,12 +70,20 @@ jobs:
       - name: Install binutils and rustfilt
         run: cargo install cargo-binutils rustfilt
 
+      - name: Build for tests
+        uses: actions-rs/cargo@v1
+        env:
+          RUSTFLAGS: "-C instrument-coverage"
+          LLVM_PROFILE_FILE: "iota-%m.profraw"
+        with:
+          command: test
+          args: --tests --package iota-sdk --all-features --no-run -- --include-ignored
```

Comment:
Our of curiosity, can you elaborate a bit on this new step?

Reply:
I originally added it simply to try to separate building and running the tests, to see if that would help identify the problem. Then I thought it might be smart to hold off on starting the private tangle until the build is done, otherwise it would run the entire 20+ minutes.

Reply:
Ok, I doubt the private tangle running in the meantime would be an issue though, it doesn't do anything if there is no activity